model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/stencils/add_extra_diffusion_for_normal_wind_tendency_approaching_cfl.py

```diff
@@ -50,26 +50,24 @@ def _add_extra_diffusion_for_normal_wind_tendency_approaching_cfl(
         (z_w_con_c_full, ddqz_z_full_e, ddt_vn_apc, cfl_w_limit), wpfloat
     )
 
-    w_con_e = where(
-        # TODO(havogt): my guess is if the second condition is `True`, then `(levelmask | levelmask(Koff[1]))` is also `True`
```

Comment:
Can you move this comment down to line 59? Unless someone can improve the "my guess" part. @OngChia